### PR TITLE
[Doc] Refactor simt

### PR DESCRIPTION
**Pull request by neozhaoliang**

### CODE REVIEW

File: docs/lang/articles/reference/simt.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 5
+---
+
+# SIMT intrinsics
+
+For CUDA backend, Taichi now supports warp-level and block-level intrinsics that
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/reference/simt.md:7]:
```suggestion
For the CUDA backend, Taichi supports warp-level and block-level intrinsics, which
```

File: docs/lang/articles/reference/simt.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+|`ti.simt.warp.shfl_up_f32`  | `__shfl_up_sync`  |
+|`ti.simt.warp.shfl_down_i32`| `__shfl_down_sync`|
+|`ti.simt.warp.shfl_down_f32`| `__shfl_down_sync`|
+|`ti.simt.warp.shfl_xor_i32` | `__shfl_xor_sync` |
+|`ti.simt.warp.match_any`    | `__match_any_sync`|
+|`ti.simt.warp.match_all`    | `__match_all_sync`|
+|`ti.simt.warp.active_mask`  | `__activemask`    |
+|`ti.simt.warp.sync`         | `__syncwarp`      |
+
+Please refer to our [API docs](https://docs.taichi.graphics/api/taichi/lang/simt/warp/#module-taichi.lang.simt.warp)
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/reference/simt.md:30]:
```suggestion
See [Taichi's API reference](https://docs.taichi-lang.org/api/taichi/lang/simt/warp/#module-taichi.lang.simt.warp)
```

File: docs/lang/articles/reference/simt.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+for more information on each function.
+
+Here is an example to perform data exchange within a warp in Taichi:
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/reference/simt.md:33]:
```suggestion
Here is an example of performing data exchange within a warp in Taichi:
```

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]
LGTM. Minor editorial changes.

**Review by yuanming-hu — APPROVED** [review]
LGTM! Thank you!

**Review by Vissidarte-Herman — APPROVED** [review]
LGTM

**Pull request merged**